### ISSUE TITLE
Version argument missing

### ISSUE THREAD

**Issue by joqim** [openai/openai-python#295]
### Describe the feature or improvement you're requesting

CLI is missing a version argument. 

openai --version would be a nice improvement

**Comment by joqim** [comment]
@FizzyAgent thank you for looking into it. I wanted to test this locally but couldn't find necessary documentation.

Is there any way you can guide me?

**Comment by FizzyAgent** [comment]
@joqim here's how you can test it locally:

1. clone my forked repo and setup your choice of environment (conda, venv, etc)

2. to build locally, I used pip
```
pip install .
```

3. make sure you are running the correct openai package
```
which openai
# should show directory to your environment, else try uninstalling and rebuilding
```

4. run the openai command
```
openai --version
```

**Comment by hallacy** [comment]
Marking this fixed since the #298 has been merged in
